tests/Behat/FeatureContext.php

```diff
@@ -187,6 +187,82 @@ protected function getLangcodeByName(string $language_name): string {
     throw new \Exception("Language name '$language_name' is not valid.");
   }
 
+  /**
+   * Visit a edit page by node title.
+   *
+   * @When I visit node :arg1 edit page
+   */
+  public function iVisitNodeEditPage($arg1) {
+    $node = $this->getNodeByTitle($arg1);
+    $this->visitPath("/node/{$node->id()}/edit");
+  }
+
+  /**
+   * Checks, that form field has specified value.
+   *
+   * @Then the :field field contains :value
+   */
+  public function theFieldContains($field, $value) {
+    $node = $this->getSession()->getPage()->findField($field);
+    $actual = $node->getValue();
+
+    if (strpos($actual, $value) === FALSE) {
+      throw new \Exception(sprintf('Field %s does not contain %s.', $field, $value));
+    }
+  }
+
+  /**
+   * Attempts to find a button in a table row containing giving text.
+   *
+   * @When I press :button in the :rowText row
+   */
+  public function iPressInTheRow($button, $rowText) {
+    $page = $this->getSession()->getPage();
+    $rows = $page->findAll('css', 'tr');
+
+    if (empty($rows)) {
+      throw new \Exception(sprintf('No rows found on the page %s', $this->getSession()->getCurrentUrl()));
+    }
+
+    /** @var \Behat\Mink\Element\NodeElement $row */
+    foreach ($rows as $row) {
+      if (strpos($row->getText(), $rowText) !== FALSE) {
+        break;
+      }
+    }
+
+    if ($row === NULL) {
+      throw new \Exception(sprintf('Row "%s" not found', $rowText));
+    }
+
+    $button_element = $row->findButton($button);
+
+    if ($button_element === NULL) {
+      throw new \Exception(sprintf('Button "%s" not found in row %s', $button, $rowText));
+    }
+
+    $button_element->press();
+  }
+
+  /**
+   * Assert option is selected.
+   *
+   * @When :option should be selected for :field select
+   */
+  public function shouldBeSelectedForSelect($option, $field) {
```

Comment:
**IF** we wont use this let's remove it

Reply:
I use this method in the refactored Behat statements

Reply:
Don't we have this already from the Drupal extension?